sparse_strips/vello_hybrid/shaders/sparse_strip_renderer.wgsl

```diff
@@ -129,3 +130,16 @@ fn unpack_alphas_from_channel(rgba: vec4<u32>, channel_index: u32) -> u32 {
         default: { return rgba.x; }
     }
 }
+
+// Polyfills `unpack4x8unorm`.
+//
+// Downlevel targets do not support native WGSL `unpack4x8unorm`.
+fn unpack4x8unorm(rgba_packed: u32) -> vec4<f32> {
```

Comment:
WebGL2 doesn't support `unpack4x8unorm`. Needs a polyfill.

Reply:
Does this have a measurable performance cost cross-platform? I don't know if this is used on a hot path

Reply:
> Does this have a measurable performance cost cross-platform?

Probably measurable, but likely small. We don't have benchmarks yet for Vello Hybrid, so it's hard to say.

This is run once per pixel for each wide tile command. So, it could be considered a hot path. 

We could consider preprocessing the shaders to allow directives like `#ifdef webgl /* add polyfill */ #endif`, but I think that's out of scope at this stage and better to be looked at in the future (?).

Reply:
Might be worth filing a tracking issue for this on our end. Also might be worth bringing this up with upstream wgpu.

Reply:
Good idea!

I created an issue (https://github.com/linebender/vello/issues/876). 

---

As per WGPU's README, WebGL support is "best effort" and I imagine an API like `unpack4x8unorm` which is available in OpenGL 4 but not 3 (which WebGL2 uses) could fall into the bucket of "best effort" but not full support. 

I suspect WGPU would need to support a `BROWSER_WEBGL` backend like its [`BROWSER_WEBGPU` backend](https://github.com/gfx-rs/wgpu/blob/ccaf9a84f8118f4a991e0046dbaecef4ea1c6950/wgpu-types/src/lib.rs#L249), which seems onerous.

In any case, I'll investigate existing WGPU WebGL issues and WGPU code separately later this week.

Reply:
See https://github.com/gfx-rs/wgpu/blob/ccaf9a84f8118f4a991e0046dbaecef4ea1c6950/wgpu-types/src/lib.rs#L147-L148
That is, it's expected to support WebGL

Reply:
I guess! The readme conflicts with that expectation, but I think you've found the culprit here: https://github.com/gfx-rs/wgpu/issues/4525 🙏